src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -14,10 +14,17 @@ public class SchemaBuilder
     {
         private readonly IDictionary<string, IGraphType> _types = new Dictionary<string, IGraphType>();
 
-        public IDependencyResolver DependencyResolver { get; set; } = new DefaultDependencyResolver();
+        public IServiceProvider ServiceProvider { get; set; } = new DefaultServiceProvider();
 
         public TypeSettings Types { get; } = new TypeSettings();
 
+        public IDictionary<string, Type> Directives { get; } = new Dictionary<string, Type>
+        {
+            {"deprecated", typeof(DeprecatedDirectiveVisistor)}
+        };
+
+        private List<IVisitorSelector> Visitors { get; } = new List<IVisitorSelector>();
```

Comment:
`VisitorSelectors` or just `Selectors` property name suits better. I was a little confused when I saw that visitors has other visitors inside.